sas-format/plugin.json

```diff
@@ -0,0 +1,14 @@
+{
+    "id" : "sas-format",
+    "version" : "0.0.1",
+
+    "meta" : {
+        "label" : "SAS format provided by Pandas",
+        "description" : "Format supporting .sas7bdat files from IBM SAS. This implementation is provided by Pandas and may be slower than the light one provided with DSS.",
```

Comment:
- _Format supporting **reading** .sas7bdat files…_
- Include a rationale of why this plugin can be useful, e.g. _… but supports more options_